isb_web/main.py

```diff
@@ -262,7 +265,10 @@ def thing_list(
     total_records, npages, things = sqlmodel_database.read_things_summary(
         session, offset, limit, status, authority
     )
-
+    properties = {
+        "authority": authority or "None"
+    }
+    analytics.record_analytics_event(AnalyticsEvent.THING_LIST, user_agent, x_forwarded_for, "http://isamples.org", properties)
```

Comment:
The hardcoded "http://isamples.org" is obviously wrong -- need to figure out the correct way to get that out of FastAPI when running behind nginx.

Reply:
But, in theory, we'd have a line like this for every API call that we expose.

Reply:
All seems quite straight forward.

The service URL should be available in `Request.url`, I think it has the external url that was called.